pynest/nest/tests/test_rate_neuron.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+#
+# test_rate_neuron.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+# This script tests the rate_neuron in NEST.
+
+import nest
+import unittest
+import math
+import numpy as np
+
+
+@nest.check_stack
+class RateNeuronTestCase(unittest.TestCase):
+    """Check rate_neuron"""
+
+    def setUp(self):
+        # test parameter
+        self.rtol = 0.05
+
+        # neuron parameters
+        self.neuron_params = {'mean': 1.5, 'std': 0.5, 'tau': 5.}
+
+        # simulation parameters
+        self.simtime = 1000.
+        self.dt = 0.1
+        self.tstart = 10. * self.neuron_params['tau']
+
+        nest.set_verbosity('M_WARNING')
+        nest.ResetKernel()
+        nest.SetKernelStatus(
+            {'resolution': self.dt, 'use_wfr': False, 'print_time': True})
+
+        # set up rate neuron and devices
+        self.rate_neuron = nest.Create(
+            'lin_rate_ipn', params=self.neuron_params)
```

Comment:
This test only includes the input noise neuron. It would be straightforward to extend this to test the output noise neuron as well. Wouldn't this be a good idea?